.travis.yml

```diff
@@ -12,22 +12,22 @@ install: true
 go_import_path: github.com/ONSdigital/rm-survey-service
 
 script:
+  - docker login -u "${DOCKER_GCP_USERNAME}" -p "${DOCKER_GCP_PASSWORD}" "${DOCKER_GCP_REGISTRY}";
```

Comment:
docker login is called again below, I don't think it needs to be called here.

Reply:
needs to pull gcp image in integration tests

Reply:
👍 makes sense, would be worth mentioning that change in the PR description.